Menu/SubMenu/Unit/unit_menu.gd

```diff
@@ -16,8 +16,6 @@ func _on_button_pressed(button: int):
 	match button:
 		1:
 			get_parent().load_scene_home("res://Menu/SubMenu/Unit/Display/view_units.tscn")
-		2:
-			get_parent().load_scene_home("res://Menu/SubMenu/Unit/Squad/manage_squad.tscn")
```

Comment:
We do have the manage squad scene now, so this line should still be here.

Reply:
OH i'll see what happend and compare to your previous commits. Fix it soon

Reply:
Hello @aMytho ! I think it's fixed so you can review it/merge the PR.

If you have any idea on where to work next, ping me in an issue ! I'll try my best to work on it.